samples/13.core-bot/deploymentTemplates/template-with-new-rg.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Maybe this document could be empty, since probably there is another task to migrate them all together